homeassistant/components/gotify/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "gotify",
```

Comment:
The name from the manifest is automatically used as title, only set the title in strings.json if it should be translated or if it is different from the manifest.
```suggestion
```